kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -93,6 +94,8 @@
     private static final Logger logger = LoggerFactory.getLogger(MarshallerFactory.class);
 
     private static final boolean STRICT_ID_FORMAT = Boolean.parseBoolean(System.getProperty(KieServerConstants.KIE_SERVER_STRICT_ID_FORMAT, "false"));
+    private static final String FIELDS = "fields";
+    private static final String NOT_NULL = "not_null";
```

Comment:
this is nitpicking.... what about calling the parameter filter : none | not_null or something of the sort. wdyt ?

Reply:
you mean none being equivalent to returning everything?

Reply:
NA